Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs

```diff
@@ -324,5 +326,54 @@ public void SpeakText(string text)
             LogSpeech(text);
             #endif
         }
+
+        /// <summary>
+        /// RE for stripping HTML
+        /// </summary>
+        //static Regex _htmlRegex = new Regex("<.*?>", RegexOptions.SingleLine);
+        private static readonly Regex _stripTagsRegex = new Regex(@"</?\w+((\s+\w+(\s*=\s*(?:"".*?""|'.*?'|[^'"">\s]+))?)+\s*|\s*)/?>", RegexOptions.Singleline);
```

Comment:
Please follow variable naming convention in this file. I don't believe variables start with _.
Example: stripTagsRegex